RLTest/__main__.py

```diff
@@ -356,23 +356,17 @@ def __init__(self):
             # when running on existing env we always reuse it
             self.args.env_reuse = True
 
-        Defaults.module = self.args.module
-        module_args = None
-
+        # unless None, they must match in length
         if self.args.module_args:
             len_module_args = len(self.args.module_args)
             modules = self.args.module
             if type(modules) == list:
                 if (len(modules) != len_module_args):
                     print(Colors.Bred('Using `--module` multiple time implies that you specify the `--module-args` in the the same number'))
                     sys.exit(1)
-                module_args = []
-                for args in self.args.module_args:
-                    module_args += [' '.join(args) if args else '']
-            else:
-                module_args = ' '.join(self.args.module_args)
 
-        Defaults.module_args = module_args
+        Defaults.module = fix_modules(self.args.module)
+        Defaults.module_args = fix_modulesArgs(Defaults.module, self.args.module_args)
```

Comment:
Fixed again?

Reply:
`Defaults`, `Env`, and `StandardEnv` have independent module/args handling mechanisms (`Env` does consider `Defaults`, `StandardEnv` does not). So we need to fix those args in every constructor seperately.